rest-tests/src/test/java/io/syndesis/qe/RestTestHooks.java

```diff
@@ -37,6 +43,27 @@ public void afterTest() {
         stepStorage.flushStepDefinitions();
         log.debug("Flushed steps from steps storage");
         SampleDbConnectionManager.closeConnections();
+
+        if (TestConfiguration.isDeloreanEnvironment()) {
+            //delete all integrations and connections after the test. Only for Delorean since it doesn't support TEST_SUPPORT env
+            IntegrationsEndpoint integrationsEndpoint = new IntegrationsEndpoint();
```

Comment:
use `@Autowired`

Reply:
correct. It can be used there now. I didn't use it before because the bean cannot be used in a before static method.